Correct English (minor) and some improvements

peterhauke opened the pull request:
Fix "double square brackets" in links

shawna-slh commented:
Thanks much @peterhauke !

I should be able to get to this in the next couple days.

One thing to change back, though, is the double brackets. We actually use those for updating document names and for translations.

From the [WAI resource template](https://github.com/w3c/wai-resource-template/blob/main/content/index.md):
>* ```[[Exact Title of Page on WAI website]](/permalink/to/page/)```
>* ```[WAI website but not exact title](/permalink/to/page/)```
>* ```[link off WAI website](https://example.com)```

peterhauke commented:
Thanks @shawna-slh I checked those links and changed them back. I also checked the other files where I replaced the double square brackets, but they were OK as they linked to external resources.
